src/2-run-ipfs-on-a-vps/index.md

```diff
@@ -0,0 +1,79 @@
+---
+baseurl: ..
+template: tmpl/layouts/post.html
+breadcrumbs:
+  - {name: "2-run-ipfs-on-a-vps", link: "./" }
+id: 2-run-ipfs-on-a-vps
+date: 2015-11-02
+title: Run IPFS latest on a VPS
+author: Kyle Drake
+collection: posts
+---
+
```

Comment:
i would LOVE to have an https://asciinema.org of this whole thing, so people can watch it, and follow along.

can use this package https://github.com/jbenet/asciinema-selfhost to re-host it to ipfs so you dont have to rely on asciinema. see https://github.com/jbenet/asciinema-selfhost#show-me (it can be iframe embeded, just github issues doesnt let us)